sydent/http/httpclient.py

```diff
@@ -60,7 +60,7 @@ def get_json(self, uri, max_size=None):
         try:
             # json.loads doesn't allow bytes in Python 3.5
             json_body = json_decoder.decode(body.decode("UTF-8"))
-        except Exception as e:
```

Comment:
I left this-I wasn't quite sure how the `e` was functioning (other than storing an exception if thrown?)

Reply:
The `e` is just there to create a variable which points to the actual instance of the `Exception` you caught. Useful if you want to actually _do_ something with the specific exception (programmatically inspecting it, etc.)